apps/marssnow.yml

```diff
@@ -0,0 +1,21 @@
+name: Marssnow
```

Comment:
Please delete this file, apps related stuff should be in the app folder not in the root.

Reply:
i'm soo sorry ,it's delete